### PR TITLE
doc: add a guide on running aklite locally

### PR DESCRIPTION
**Pull request by mike-sul**
An initial version of a guide on running aklite on a local host (Linux-based).

Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

File: how-to-run-locally.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+## Work dir setup
+
+`mkdir local-device-001 && cd local-device-001`
+
+### Optionally setup docker-compose and docker for your setup
+#### Docker-compose
+```
+python3 -m venv venv
+python3 -m pip install --upgrade pip
+python3 -m pip install docker-compose
```
Comment (doanac) [inline how-to-run-locally.md:13]: These python invocations should use the venv right?

Comment (mike-sul) [reply how-to-run-locally.md:14]: yep, fixed.

**Comment by doanac** [comment]
looks good to me. Since @ldts has gone through the steps, i'll give him the final approval.

**Comment by mike-sul** [comment]
@doanac We agreed with @ldts that I would update it after he went through it. So, I updated the doc/PR according to Jorge experience/feedback.

**Review by ldts — APPROVED** [review]
looks good to me. I'll probably expand it as I find more use cases we can cover. thanks!

**Pull request merged**